test/integration_test/lib/input/test_check_headers.py

```diff
@@ -7,9 +7,10 @@
 
 class TestCheckHeaders(object):	
 	
-	headers_file='./test/integration_test/data/headers_for_check.json'
+	in_headers_file='./test/integration_test/data/headers_for_check.json'
 	unwanted_uid_file='./test/integration_test/data/unwanted_uid.txt'
 	uid_map_file='./test/integration_test/data/thread_uid_map.json'
+	out_headers_file='./etc/.tmp/integration_test/lib/input/check_headers/check_headers_output.json'
```

Comment:
this van be `expected_headers_file` and `in_headers_file` can be replaced with `headers_file`